src/wasm/wasm-s-parser.cpp

```diff
@@ -649,12 +649,22 @@ Expression* SExpressionWasmBuilder::makeExpression(Element& s) {
   const char *str = id.str;
   const char *dot = strchr(str, '.');
   if (dot) {
+    // memory.
+    if (str[0] == 'm') {
+      // memory.size
+      if (str[7] == 's') return makeHost(s, HostOp::MemorySize);
+      // memory.grow
+      if (str[7] == 'g') return makeHost(s, HostOp::MemoryGrow);
+      abort_on(str);
+    }
+
```

Comment:
Most of the changes so far are naive find-and-replaces with case sensitivity. I've fixed a few spots where I found that it made things incorrect. e.g. `memory.size()` vs `memory_size()` but there might be others.

This was one such spot, where I needed to change the parser itself because it assumed that anything with a dot was `i32|i64|f32|f64` operations.